frontend/packages/pipelines-plugin/src/components/import/pipeline/pipeline-template-utils.ts

```diff
@@ -79,3 +106,45 @@ export const createPipelineRunForImportFlow = async (pipeline: Pipeline): Promis
   };
   return submitStartPipeline(pipelineInitialValues, pipeline);
 };
+
+export const updatePipelineForImportFlow = async (
+  pipeline: Pipeline,
+  template: Pipeline,
+  name: string,
+  namespace: string,
+  gitUrl: string,
+  gitRef: string,
+  gitDir: string,
+  dockerfilePath: string,
+): Promise<Pipeline> => {
+  let updatedPipeline = _.cloneDeep(pipeline);
+
+  if (!template) {
+    updatedPipeline.metadata.labels = _.omit(
+      updatedPipeline.metadata.labels,
+      'app.kubernetes.io/instance',
+    );
+  } else {
+    const runtimeLabel = 'pipeline.openshift.io/runtime';
+    if (template.metadata?.labels[runtimeLabel] !== pipeline.metadata?.labels[runtimeLabel]) {
+      updatedPipeline = _.cloneDeep(template) as Pipeline;
```

Comment:
```suggestion
      updatedPipeline = _.cloneDeep(template);
```
Shouldn't need to do this